uint8 id
time timestamp

vector2 position
vector2 velocity
vector2 acceleration

float32 yaw
float32 yaw_rate

float32 maximum_brake
